README.md

```diff
@@ -1,3 +1,5 @@
 # StrongholdSiege
 
 This is a game.
+
+Is it a game though. It somehow didnt have haashims added text ;;;100000
```

Comment:
What is this?

Reply:
You wont understand, its an intellectual thing.